README.md

```diff
@@ -89,13 +94,13 @@ Moonbeam nodes support multiple public chains and testnets, with the following C
 
 | Network Description                 | Chain ID    |
 | ----------------------------------- | ----------- |
-| Local parachain testnet             | 1280        |
-| Local standalone testnet            | 1281        |
-| Reserved for other testnets         | 1282 - 1283 |
+| Local Parachain TestNet             | 1280        |
+| Local Standalone Node               | 1281        |
```

Comment:
In https://substrate.dev/rustdocs/v2.0.0/frame_support/traits/trait.Randomness.html and the following discussion, @crystalin reminded me that standalone means aura rather than parachain. It is the network that is standing alone from any relay chain as opposed to an individual node standing alone.
```suggestion
| Local Standalone TestNet            | 1281        |
```